presto-password-authenticators/src/main/java/io/prestosql/plugin/password/salesforce/SalesforceConfig.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.password.salesforce;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.log.Logger;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.NotNull;
+
+import java.util.HashSet;
+import java.util.Locale;
+import java.util.Set;
+
+public class SalesforceConfig
+{
+    private static final Logger log = Logger.get(SalesforceConfig.class);
+
+    protected static final String LOGINURL = "https://login.salesforce.com/services/Soap/u/";
+    protected static final String APIVERSION = "46.0";
```

Comment:
(nit) Add underscores to the names to the spell checker in the ID doesn't complain